tests/unit/core/TestThemeInstall.js

```diff
@@ -11,25 +9,6 @@ describe(__filename, () => {
     }).toThrowError('Invalid theme action requested');
   });
 
-  it('sets-up the event for previews', () => {
```

Comment:
I wonder if we need a test like this for the install action. That doesn't seem to be tested anywhere.

Reply:
hmm..i do see tests in InstallButton (eg: `calls installTheme when clicked`). I am not sure if this is enough. I'll take another look tho..